admin_guide/securing_builds.adoc

```diff
@@ -0,0 +1,168 @@
+= Securing Builds by Build Strategy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Builds in OpenShift are run in privileged containers that have access to the Docker daemon socket. As a security
+measure, it is recommended to limit who can run builds and the strategy that is used for those builds. Docker and Custom
+builds are inherently less safe than Source builds, given that they can execute any code in the build with
+potentially full access to the node's Docker socket.
+
+By default, project administrators (`admin` role) and project editors (`edit` role) are granted
+permission to use all build strategies (Docker, Source-to-Image, and Custom).
+
+Who can build with what build strategy can be controlled via an
+link:../architecture/additional_concepts/authorization.html[authorization policy].
+Each build strategy has a corresponding build subresource. Granting permission to `create` on the
+build subresource will allow the user to create builds of that type.
+
+.Build strategy subresources
+[cols="1,1",options="header"]
+|===
+
+|Strategy |Resource
+
+|Docker
+|builds/docker
+
+|Source-To-Image
+|builds/source
+
+|Custom
+|builds/custom
+
+|===
+
+
+
+== Disabling a build strategy globally
+To prevent access to a particular build strategy globally, login as cluster administrator and edit
+each of the default roles:
+
+----
+$ oadm edit clusterrole admin
+$ oadm edit clusterrole edit
+----
+
+For each role, remove the line that corresponds to the resource of the strategy to disable:
+
+.Disable the Docker build strategy
+=====
+
+----
+kind: ClusterRole
+metadata:
+  name: admin
+...
+
+rules:
+- attributeRestrictions: null
+  resources:
+  - builds/custom
+  - builds/docker <1>
+  - builds/source
+  - pods/exec
+  - pods/portforward
+
+...
+
+----
+<1> Delete this line to disable Docker builds globally
+
+=====
+
+Alternatively, the policy can be updated via non-interactive commands.
+
+.Disable the Docker build strategy using non-interactive commands
```

Comment:
is this worth including?  since we're not demonstrating a specific feature related to builds or even roles, it feels out of place.  "scriptable stuff can be scripted" isn't something we should explain in each section where we find ourselves doing editing.